homeassistant/components/recorder/statistics.py

```diff
@@ -1486,7 +1503,11 @@ def statistic_during_period(
         tail_start_time: datetime | None = None
         tail_end_time: datetime | None = None
         if end_time is None:
-            tail_start_time = now.replace(minute=0, second=0, microsecond=0)
+            tail_start_time = _last_statistic(session, Statistics, metadata_id)
+            if tail_start_time:
+                tail_start_time -= timedelta(hours=1)
```

Comment:
I don't think this is correct

Reply:
Probably could use a test to see how this behaves if HA has been shutdown for 3 hours, 2 hours, and 1 hour